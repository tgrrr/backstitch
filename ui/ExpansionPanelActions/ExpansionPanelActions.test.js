import React from 'react'
import renderer from 'react-test-renderer'
import { ExpansionPanelActions } from '.'

it('ExpansionPanelActions: default', () => {
    const component = renderer.create(
        <ExpansionPanelActions>
            <>
                ExpansionPanelActions
            </>
        </ExpansionPanelActions>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
