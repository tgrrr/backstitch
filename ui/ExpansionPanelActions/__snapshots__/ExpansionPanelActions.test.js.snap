// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpansionPanelActions: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Expansionpanelactions "
    data-testid="Expansionpanelactions"
  >
    <div
      className="MuiExpansionPanelActions-root MuiExpansionPanelActions-spacing"
    >
      ExpansionPanelActions
    </div>
  </div>
</div>
`;
